paris-material/src/main/java/com/airbnb/paris/material/proxies/MaterialButtonProxy.kt

```diff
@@ -0,0 +1,161 @@
+package com.airbnb.paris.material.proxies
+
+import android.content.res.ColorStateList
+import android.graphics.PorterDuff
+import android.graphics.drawable.Drawable
+import android.util.Log
+import android.util.TypedValue
+import androidx.annotation.Px
+import androidx.annotation.StyleRes
+import com.airbnb.paris.annotations.AfterStyle
+import com.airbnb.paris.annotations.Attr
+import com.airbnb.paris.annotations.Styleable
+import com.airbnb.paris.material.R
+import com.airbnb.paris.material.R2
+import com.airbnb.paris.material.utils.ViewUtils
+import com.airbnb.paris.proxies.BaseProxy
+import com.airbnb.paris.styles.Style
+import com.google.android.material.button.MaterialButton
+
+@Styleable("Paris_MaterialButton")
+class MaterialButtonProxy(view: MaterialButton) : BaseProxy<MaterialButtonProxy, MaterialButton>(view) {
+
+    private var mColors: ColorStateList? = null
+    private val defaultTextAppearance:Int = 16974317 //todo this might not be the default for every android version or phone
+    private var mTextAppearance = defaultTextAppearance
```

Comment:
This project doesn't follow the `m` prefix convention. You can call these `colors` and `textAppearance`.